std/range/package.d

```diff
@@ -2887,13 +2905,37 @@ private:
     UT _value;
 
 public:
-    @property inout(T) front() inout { return _value; }
-    @property inout(T) back() inout { return _value; }
+    ///
+    @property inout(T) front() inout
+    {
+        return _value;
+    }
+
+    ///
```

Comment:
Is this the common way to document these?
I thought it's preferred to just explicitly document these methods and their meaning is intrinsic.